<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheBox</class>
 <widget class="QMainWindow" name="TheBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>961</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>581</y>
      <width>833</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QFormLayout" name="LayoutInfinito">
       <item row="0" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>X:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rX">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>240.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Y:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rY">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>431.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Vx:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rVx">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Vy:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rVy">
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>ax:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rax">
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>ay:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="ray">
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>-1000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="ZoomPlus">
               <property name="text">
                <string>Zoom+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ZoomMinus">
               <property name="text">
                <string>Zoom-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="Iniciar">
             <property name="text">
              <string>Iniciar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Detener">
             <property name="text">
              <string>Detener</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Ingresar">
             <property name="text">
              <string>Ingresar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Left">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="Up">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Right">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>95</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>ms: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rClockMs">
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>T: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="rPeriodo">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>660</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>damage</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
